app/assets/stylesheets/dashboard.css

```diff
@@ -216,6 +216,20 @@
   margin-right: 5px;
 }
 
+/* Issue #5031 (daddyjab):
+   Revised to add border of
+   offsetting color to Twitter buttons */
+.dashboard .social-media-btn #follow-button {
+  border: 1px solid #00aced;
+}
+
+/* Issue #5031 (daddyjab):
```

Comment:
Here as well :) 
Comments are always nice to understand the code but i feel it is redundant here !